### ISSUE TITLE
Window size is not saved

### ISSUE THREAD

**Issue by galad87** [ZeeZide/Shrugs#8]
Open Shrugs, resize the window. Quit it, reopen it, window size is back to the default value.

**Comment by helje5** [comment]
To get full window restoration, go to the Mac preferences, disable "Close windows when quitting an app".

<img width="395" alt="Screenshot 2020-06-13 at 11 06 13" src="https://user-images.githubusercontent.com/7712892/84564943-e6944400-ad65-11ea-8388-113389e1c289.png">

Maybe there should be a setting in Shrugs to ignore/override the system here, like other apps do (e.g. I think Mail.app does).

**Comment by galad87** [comment]
Usually apps on macOS save the windows size even if that option is enabled. For example: Safari, Finder, Photos, Mail, Messages, Books, Notes… oh well I guess everyone does.

**Comment by helje5** [comment]
You are right, the last's window size should be preserved regardless. Will fix that.

But the option also does disables window restoration (just tried w/ Safari, I think Mail.app is the only app I've seen which overrides this) which is what most people really want, I think.
